scripts/migrate_to_rapidpro/async_archive_chats_step2.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+import csv
+import os
+import sys
+
+import aiohttp
+from retry_requests import request
+from six.moves import urllib_parse
+
+CONCURRENCY = 20
+TURN_URL = "https://whatsapp.praekelt.org"
+RAPIDPRO_URL = "https://rapidpro.prd.momconnect.co.za"
+
+
+async def process_conversation(session, row, target):
+    whatsapp_id = str(row["owner"]).replace("+", "")
+    last_message_id = row["last_message_id"]
+
+    url = None
+    if row["action"] == "ARCHIVE":
+        url = urllib_parse.urljoin(TURN_URL, f"v1/chats/{whatsapp_id}/archive")
+        headers = {
+            "Authorization": f"Bearer {os.environ['TURN_TOKEN']}",
+            "Accept": "application/vnd.v1+json",
+            "Content-Type": "application/json",
+        }
+        data = {"before": last_message_id, "reason": "Bulk archived"}
```

Comment:
For archiving, should we not also clear the `wait_for_helpdesk`?

Reply:
For archiving it will write two rows to the csv for step 2, one for the turn archive and one to update rapidpro:
https://github.com/praekeltfoundation/ndoh-hub/pull/363/files#diff-bf9376422573236eba1a43847404ade403ac12d05289f558772941b36ed93adfR85-R99